src/StreamService/Startup.cs

```diff
@@ -0,0 +1,51 @@
+using LT.DigitalOffice.Kernel.Configurations;
+using LT.DigitalOffice.Kernel.Middlewares.ApiInformation;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.StreamService
+{
+    public class Startup : BaseApiInfo

```

Comment:
are you sure this class should look like this?

Reply:
fixed